quiffen/__init__.py

```diff
@@ -18,4 +18,9 @@
 from quiffen.core.qif import ParserException, Qif, QifDataType
 from quiffen.core.security import Security
 from quiffen.core.split import Split
-from quiffen.core.transaction import Transaction, TransactionLike, TransactionList
+from quiffen.core.transaction import (
+    Transaction,
+    TransactionConfig,
+    TransactionLike,
+    TransactionList,
+)
```

Comment:
This seems to have code from other open PRs. Please separate this so it only contains the relevant code.

Reply:
progress is a tqdm or ducktype compatible progress  bar. For large imports having the progress bar is IMHO a very important feature and the implementation is quite non invasive.

Reply:
Then it needs to have a type hint as such, at the very least. Again, this is another feature that needs discussion, as it has long-term implications for me in terms of maintenance.